modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DocumentLibrary/DocumentLibrary.es.js

```diff
@@ -209,12 +211,27 @@ const Main = ({
 	onFocus,
 	placeholder,
 	readOnly,
+	valid,
 	value = '{}',
 	...otherProps
 }) => {
 	const {portletNamespace} = usePage();
 	const [currentValue, setCurrentValue] = useState(value);
 
+	const getErrorMessages = (errorMessage, isSignedIn) => {
+		const errorMessages = [errorMessage];
+
+		if (!isSignedIn) {
+			errorMessages.push(
+				Liferay.Language.get(
+					'you-need-to-be-signed-in-to-edit-this-field'
+				)
+			);
+		}
+
+		return errorMessages.join(' ');
```

Comment:
@carolmariaabb, just one question, what is the purpose of this `join`?

Reply:
Hi @natocesarrego,

I thought that if the field is required it would be interesting for the user to also see this error message. 

So I created an array of strings and the `join` creates a new string by concatenating all the messages in this array separated by the specified character.